### ISSUE TITLE
Don't hear anything in mobile Safari

### ISSUE THREAD

**Issue by Joshfindit** [naomiaro/waveform-playlist#76]
iPad Air 2 with iOS 10.2

Everything seems to work, the UI is responsive to touch, the volume updates, etc, but there is no sound on the examples.

**Comment by naomiaro** [comment]
@Joshfindit  Do you experience this on every example and even after refreshing the page? I'm trying on a couple iPads now and I can hear sound. The first time I loaded minimal example I got a squashed track with really bad sound, however when I reloaded it was the normal track I'd expect. I'm wondering if there's some kind of caching issue there.

**Comment by naomiaro** [comment]
Webaudio also seems to be affected by the mute button where as other sounds might not be:

https://github.com/goldfire/howler.js/issues/753

**Comment by naomiaro** [comment]
Also maybe I'll check into this

- https://stackoverflow.com/questions/12517000/no-sound-on-ios-6-web-audio-api

**Comment by naomiaro** [comment]
I'm able to hear the examples still on an iPhone 6 with the mute switch turned on, so I guess that article is outdated.

**Comment by Joshfindit** [comment]
Unfortunately it seems like it's destined to be a phantom issue: when I tried it just now it was fine.
Tried closing it and reopening after mute was turned on (certainly possible I had it on mute as I do regularly since music and video play fine with mute on), but with the iPad muted it still played.

It could also have needed a reboot, I've done at least once since the issue.

Thanks for the great roundup of tips, hopefully it happens again now that I know what to look for.
